#!/usr/bin/env python
PACKAGE = "gptrajpredict"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("distance_heuristic_cost_scale", double_t, 0, "gridmap_distance_heuristic",    3.0, 0,   100)

gen.add("rollover_cost_scale", double_t, 0, "gridmap_rollover_cost_scale",    2.0, 0,   100)
gen.add("kernel_dist_cost_scale", double_t, 0, "gridmap_kernel_dist_cost_scale",    0.05, 0,   100)
gen.add("prediction_diff_cost_scale", double_t, 0, "prediction_diff_cost_scale",    27.0, 0,   100)



gen.add("set_goal",   bool_t,   0, "set_goal",  False)
gen.add("goal_x", double_t, 0, "goal_x",    13.5, -25,   30)
gen.add("goal_y", double_t, 0, "goal_y",    25.0, -25,   30)


exit(gen.generate(PACKAGE, "gptrajpredict", "reconfig"))